flatgfa/src/cli/cmds.rs

```diff
@@ -299,3 +301,50 @@ pub fn gaf_lookup(gfa: &flatgfa::FlatGFA, args: GAFLookup) {
         }
     }
 }
+
+///
+#[derive(FromArgs, PartialEq, Debug)]
+#[argh(subcommand, name = "seq-import")]
+pub struct SeqImport {
+    #[argh(positional)]
+    filename: String,
+}
+
+pub fn seq_import(args: SeqImport) {
+    let vec = packedseq::import(&args.filename);
+    let _ = std::fs::remove_file(&args.filename);
```

Comment:
Are you sure you want to delete the file after reading it? It seems like it would be more predictable if we just left it alone, and the user can delete it if they want.